lib/notifications-status-view.js

```diff
@@ -0,0 +1,87 @@
+const { CompositeDisposable, Disposable } = require('atom');
+
+module.exports = class NotificationsStatusView {
+
+    constructor(statusBar, duplicateTimeDelay) {
+      this.count = 0;
+      this.duplicateTimeDelay = duplicateTimeDelay;
+      this.click = this.click.bind(this);
+      this.animationend = this.animationend.bind(this);
+      this.subscriptions = new CompositeDisposable();
+      this.render();
+      this.tile = statusBar.addRightTile({
+        item: this.element,
+        priority: 100
+      });
+      this.subscriptions.add(new Disposable(() => { this.tile.destroy(); }));
+      // TODO: uncomment next line when atom/atom#16074 is released
+      // this.subscriptions.add(atom.notifications.onDidClearNotifications(() => this.clear()));
+    }
+
+    render() {
+      this.number = document.createElement('div');
+      this.number.textContent = this.count;
+      this.number.addEventListener('animationend', this.animationend);
+      this.subscriptions.add(new Disposable(() => {
+        this.element.removeEventListener('animationend', this.animationend);
+      }));
+
+      this.element = document.createElement('a');
+      this.element.classList.add('notifications-count', 'inline-block');
+      this.tooltip = atom.tooltips.add(this.element, {title: 'Notifications'});
```

Comment:
This tooltip should be consistent with the other tooltip states.  That is, if there are existing notifications when the tile is instantiated it should show the last message, otherwise it should show "No notifications".